console/src/main/scripts/plugins/accounts/ts/accountsGlobals.ts

```diff
@@ -63,4 +93,28 @@ module HawkularAccounts {
     warning(message: string): void;
     error(message: string): void;
   }
+
+  export class Invitation implements IInvitation {
+    id:string;
+    token:string;
+    email:string;
+    organization:HawkularAccounts.IOrganization;
+    role:HawkularAccounts.IRole;
+    acceptedAt:Date;
+
+    constructor(email:string, role:HawkularAccounts.IRole) {
+      this.email = email;
+      this.role = role;
+    }
+  }
+
+  export class Role implements IRole {
```

Comment:
I need a concrete class, so that I need to create this object "on demand". It duplicates most of the code from IRole (same for the above Invitation vs. IInvitation), but I think this is a necessary evil, right?